parser/src/lexer.rs

```diff
@@ -145,6 +145,8 @@ where
     }
 }
 
+const MAX_UNICODE_NAME: usize = 88;
```

Comment:
In the future, someone browsing the codebase might come across this line and wonder why it's there. Perhaps a comment can help to clarify things. Something like:

```rust
// unicode_name2 does not expose `MAX_UNICODE_NAME`, so we replicate that constant here
```